spec/features/users/sign_up_spec.rb

```diff
@@ -108,30 +108,27 @@
     expect(page).to have_content(I18n.t('telephony.error.friendly_message.generic'))
   end
 
-  scenario 'rate limits sign-up phone confirmation attempts' do
+  scenario 'rate limits sign-up phone confirmation attempts', js: true do
```

Comment:
As far as I recall, we added `js: true` as part of debugging. Can we take it back out?

Reply:
Done.